var/spack/repos/builtin/packages/meson/package.py

```diff
@@ -16,6 +16,8 @@ class Meson(PythonPackage):
 
     maintainers = ['michaelkuhn']
 
+    version('0.61.1', sha256='f1e928978543aaf819e9fb38a6eb90f95d320dc53216f9e2c37edc6cfd5612e6')
```

Comment:
Should go straight to 0.61.2 which by a strange coincidence just got released about 57 minutes ago.